briefme_editorial/models.py

```diff
@@ -0,0 +1,103 @@
+from django.db import models
+from django.conf import settings
+
+from model_utils.models import StatusModel, TimeStampedModel
+from model_utils.fields import MonitorField
+from model_utils import Choices
+from tinymce.models import HTMLField
+
+from .constants import TEMPLATE_CHOICES
+
+
+class Publication(models.Model):
+    """
+    The model that manages all publications: daily, weekend, poll, making-of...
+    """
+
+    name = models.CharField(max_length=128)
+    description = models.TextField()
+
+    class Meta:
+        abstract = True
+
+
+    def __str__(self):
+        return self.name
+
+
+class Issue(TimeStampedModel, StatusModel):
+    """
+    The model that manages the issue of a title
+    """
+
+    STATUS = Choices("draft", "pending", "published")
+    title = models.CharField(max_length=200)
+    publication = models.ForeignKey(
+        Publication, default=settings.DEFAULT_PUBLICATION, on_delete=models.CASCADE
+    )
+    template = models.CharField(max_length=6, choices=TEMPLATE_CHOICES, default="new")
+    intro = HTMLField(blank=True, null=True)
+    outro = HTMLField(blank=True, null=True)
+    published_at = MonitorField(monitor="status", when=["published"])
+    pushed = models.BooleanField(default=False)
+    pushed_at = MonitorField(monitor="pushed", when=[True])  # CHECK brief.eco has no "when" option
```

Comment:
I don't think this is used on Brief.eco as we do not have push notifications. This could come from a copy / paste when Brief.eco was created from Brief.me.
In the future, we want to send push notifications from Brief.eco as well so I would let it in this abstract model like you did

Reply:
:ok_hand:
a794a7df59ff2d37df244f490fab9e853c0114dd